demos/bartlett1932/templates/ad.html

```diff
@@ -1,79 +1,86 @@
-			    background: white;
-			}
-		</style>
-	</head>
-	<body>
-		<div id="container-ad">
-			<div id="ad">
-				<div class="row">
-					<div class="col-xs-2">
-						<img id="adlogo" src="{{ server_location }}/static/images/logo.png" alt="Lab Logo" />
-					</div>
-					<div class="col-xs-10">
-						{% if assignmentid == "ASSIGNMENT_ID_NOT_AVAILABLE" %}
+    <head>
+        <title>Psychology Experiment</title>
+        <link rel=stylesheet href="/static/css/bootstrap.min.css" type="text/css">
+        <style>
+            body {
+                padding:0px;
+                margin: 0px;
+                background-color: white;
+                color: black;
+                font-weight: 300;
+                font-size: 13pt;
+            }
+            #adlogo {
+                float: right;
+                width: 140px;
+                padding: 2px;
+                border: none;
+            }
+            #container-ad {
+                position: absolute;
+                top: 0px; /* Header Height */
+                bottom: 0px; /* Footer Height */
+                left: 0px;
+                right: 0px;
+                padding: 100px;
+                padding-top: 5%;
+                border: 18px solid #f3f3f3;
+                background: white;
+            }
+        </style>
+    </head>
+    <body>
+        <div id="container-ad">
+            <div id="ad">
+                <div class="row">
+                    <div class="col-xs-2">
+                        <img id="adlogo" src="{{ server_location }}/static/images/logo.png" alt="Lab Logo" />
+                    </div>
+                    <div class="col-xs-10">
 
-						    <h1>Call for participants</h1>
-						    <p>
-								The XXX Lab at XXXXX University is looking for online participants
-								for a brief psychology experiment. The only requirements
-								are that you are at least 18 years old and are a fluent English
-								speaker.  The task will take XXXXX minutes and will pay XXXXX.
-						    </p>
-						    <p>
-							    Please click the "Accept HIT" button on the Amazon site
-							    above to begin the task.
-							</p>
-						{% else %}
-						    <h1>Thanks for accepting this HIT.</h1>
-						    <p>
-						    	By clicking the following URL link, you will be taken to the experiment,
-						        including complete instructions and an informed consent agreement.
-						    </p>
-						    <script>
-								function openwindow() {
-						    		popup = window.open('{{ server_location }}/consent?hit_id={{ hitid }}&assignment_id={{ assignmentid }}&worker_id={{ workerid }}&workerId={{ workerid }}','Popup','toolbar=no,location=no,status=no,menubar=no,scrollbars=yes,resizable=no,width='+1024+',height='+768+'');
-						  		}
-						    </script>
-						    <div class="alert alert-warning">
-						    	<b>Warning</b>: Please disable pop-up blockers before continuing.
-						    </div>
+                        {% if is_sandbox or debug_mode %}
```

Comment:
Just a note that, eventually, we should use proper Jinja2 template inheritance so that this can be defined once and then applied to any experiment ad just by using it as a macro / include.